src/java.base/share/classes/sun/security/ssl/SignatureScheme.java

```diff
@@ -472,6 +472,10 @@ static Map.Entry<SignatureScheme, Signature> getSignerOfPreferableAlgorithm(
             ProtocolVersion version) {
 
         PrivateKey signingKey = x509Possession.popPrivateKey;
+
+        ECParameterSpec params = x509Possession.getECParameterSpec();
```

Comment:
This 'params' variable is used for debug only.  Maybe, it could be moved to the debug log block.